delfin/drivers/dell_emc/vmax/vmax.py

```diff
@@ -95,3 +95,10 @@ def list_alerts(self, context, query_para):
         alert_model_list = unisphere_alerts.AlertHandler()\
             .parse_queried_alerts(alert_list)
         return alert_model_list
+
+    def collect_array_metrics(self, context, storage_id,
+                              interval, is_historic):
+        if not is_historic:
+            interval = perf_utils.VMAX_PERF_MIN_INTERVAL
```

Comment:
How will caller know what is interval of metric? If required, this needs to be part of response.

Reply:
caller supply interval if hisptorci collection, otherwise we have to provide minimum interval for VMAX , as VMAX will have a performamce metric in the span of 5 sec. so minimum 5 sec we have to provide to get realtime